cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -546,18 +546,18 @@ def _deserialize_gate_op(
                 arg_function_language=arg_function_language,
                 required_arg_name=None,
             )
-            invert_mask = cast(
-                List[bool],
-                arg_func_langs.arg_from_proto(
-                    operation_proto.measurementgate.invert_mask,
-                    arg_function_language=arg_function_language,
-                    required_arg_name=None,
-                ),
+            invert_mask_ = arg_func_langs.arg_from_proto(
```

Comment:
Having `invert_mask` and `invert_mask_` is pretty confusing.  Can we give one a better name?

Reply:
renamed to `parsed_invert_mask`